Helper/DeletedProductSerializer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Metrilo\Analytics\Helper;
+
+class DeletedProductSerializer extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    private function searchForParentById($parentId, $deletedProducts) {
```

Comment:
move the `private` functions after the `public` ones

Reply:
Will be fixed with next commit.